src/helpers/configHelpers.ts

```diff
@@ -22,7 +22,7 @@ import * as fs from "fs";
 import type { NeedleConfig } from "../types/needleConfig";
 import { MessageKey } from "./messageHelpers";
 
-const CONFIGS_PATH = pathResolve(__dirname, "../../configs");
+const CONFIGS_PATH = pathResolve(__dirname, "../../", process.env.CONFIGS_PATH || "configs");
```

Comment:
perhaps we should move this along with `NODE_ENV` to something like `setup.ts` file that's imported in `main.ts`, or just top level of `main.ts` that has a couple of lines for setting defaults:

```ts
process.env.NODE_ENV ??= "development"
process.env.CONFIGS_PATH ??= "configs"
```

that way, we don't even need to null check this as it would be defaulted

Reply:
I don't think this is necessary for now, but if any more env variables are added, that would be good to add.